tests/builtins/test_divmod.py

```diff
@@ -8,27 +8,6 @@ class DivmodTests(TranspileTestCase):
 class BuiltinDivmodFunctionTests(BuiltinFunctionTestCase, TranspileTestCase):
     functions = ["divmod"]
 
-    not_implemented = [
```

Comment:
Makes sense, though - if the error message has been corrected, then most of these will have been failing because the message was wrong.